<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>audioClass</class>
 <widget class="QWidget" name="audioClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>audio</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true"/>
   </item>
   <item row="0" column="2" rowspan="5">
    <widget class="QTreeView" name="treeView"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_startorstop">
     <property name="text">
      <string>start/stop</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_restart">
     <property name="text">
      <string>restart</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_next">
     <property name="text">
      <string>next</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton_previous">
     <property name="text">
      <string>previous</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QToolButton" name="toolButton_fileList">
     <property name="text">
      <string>fileList</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QToolButton" name="toolButton">
     <property name="text">
      <string>load url</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>sound</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSlider" name="horizontalSlider_sound">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>progress</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QSlider" name="horizontalSlider_progress">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="audio.qrc"/>
 </resources>
 <connections/>
</ui>
